locale/pt-br/docs/meta/topics/dependencies.md

```diff
@@ -55,19 +55,19 @@ para distribuir o trabalho que não pode ser feito assíncronamente a nível de
 
 * [Documentação](http://docs.libuv.org/)
 
-### http-parser
+### llhttp
 
-<!-- HTTP parsing is handled by a lightweight C library called http-parser. It is
+<!-- HTTP parsing is handled by a lightweight C library called llhttp. It is
 designed to not make any syscalls or allocations, so it has a very small
 per-request memory footprint.
 
-* [Documentation](https://github.com/joyent/http-parser/) -->
+* [Documentation](https://github.com/nodejs/llhttp) -->
 
 O parsing do protocolo HTTP é delegado a uma biblioteca leve, escrita em C,
```

Comment:
C and Typescript*

I sent this PR before, and one correction/suggestion was add Typescript to description. This "error" is also in other languajes.

Reply:
See: https://github.com/nodejs/nodejs.org/pull/2612#discussion_r329295703

Reply:
Yeah, well translations will need to be updated later for the Typescript part. I personally don't mind landing this as is.

Reply:
Looks good to me